internal/terraform/lang/hcl_parser.go

```
@@ -10,7 +10,13 @@ import (
 
 // ParseBlock parses HCL configuration based on tfjson's SchemaBlock
 // and keeps hold of all tfjson schema details on block or attribute level
-func ParseBlock(block *hclsyntax.Block, labels []*ParsedLabel, schema *tfjson.SchemaBlock) Block {
+func ParseBlock(tokens hclsyntax.Tokens, labels []*ParsedLabel, schema *tfjson.SchemaBlock) (Block, error) {
+	hclBlock, _ := hclsyntax.ParseBlockFromTokens(tokens)
```

Could you add a comment here as to why you are ignoring the error?